src/ptvsd/__main__.py

```diff
@@ -349,7 +349,7 @@ def escape(ch):
 
 import ptvsd.log
 ptvsd.log.to_file()
-ptvsd.log.info('Debugger successfully injected')
+ptvsd.log.info("Debugger successfully injected")
```

Comment:
I really should have known better, since I wrote the comment above. Should we add an assert here maybe? Right after `format`.

Also, I think the `print(code)` below is some debugging leftover. It's not a bad idea to log it, but it shouldn't be printed to stdout.

Reply:
Will do :)